pkg/addon/addon.go

```diff
@@ -69,8 +69,9 @@ func NewGetValuesFunc(imageName string, imageRegistryClient imageregistry.Client
 		}
 
 		for _, claim := range cluster.Status.ClusterClaims {
-			if claim.Name == "product.open-cluster-management.io" && claim.Value == "OpenShift" {
-				addonValues.IsOCP = true
+			if claim.Name == "version.openshift.io" && claim.Value != "3" {
```

Comment:
this is a danger claim, we cannot say it is OCP4 if it is not 3.

Reply:
the cluster is OCP if there is version.openshift.io clusterclaim.  only 2 options for the version, 3 or 4.x.x currently.

Reply:
I would use HasRoute as flag instead using ISOCP4. Also add a comment in this section on how we detect whether route exists

Reply:
done